python/ray/train/_internal/storage.py

```diff
@@ -614,37 +590,85 @@ def experiment_fs_path(self) -> str:
         return Path(self.storage_fs_path, self.experiment_dir_name).as_posix()
 
     @property
-    def experiment_local_path(self) -> str:
-        """The local filesystem path to the experiment directory.
+    def experiment_local_staging_path(self) -> str:
```

Comment:
Thoughts on the var naming / directory structure?

Reply:
what about putting the driver concept into the name? e.g. `driver_experiment_staging_dir`?

Reply:
Yeah, that might be clearer. I'll make the change and label this property as experimental -- users should not access it / rely on it.